File: py/minimum-number-of-steps-to-make-two-strings-anagram.py
# https://leetcode.com/problems/minimum-number-of-steps-to-make-two-strings-anagram/description/

def minSteps(s: str, t: str) -> int:
  list_s = list(s)
  for i in list_s:
    if i in t:
      t = t.replace(i, '', 1)
  return len(t)

print(minSteps('leetcode', 'practice'))

File: js/minimum-number-of-steps-to-make-two-strings-anagram.js
// https://leetcode.com/problems/minimum-number-of-steps-to-make-two-strings-anagram/
/**
 * @param {string} s
 * @param {string} t
 * @return {number}
 */
var minSteps = function(s, t) {
  let splitS = s.split('')

for (i in splitS){
if (t.includes(splitS[i])){
  t = t.replace(splitS[i], '')
}
}

return t.length
};